config/default.json

```diff
@@ -13,7 +13,7 @@
       "enabled": false
     },
     "sso": {
-      "enabled": true
+      "enabled": false
```

Comment:
do we want to turn off by default now?

Reply:
Yeah, we do, it's for local development to make it easier for the backend/db devs/